platform/ui/src/components/SplitButton/SplitButton.tsx

```diff
@@ -107,7 +105,7 @@ const SplitButton = ({
 
         setState(state => ({
           ...state,
-          primary: !isAction ? { ...item, index } : state.primary,
+          primary: !isAction && isRadio ? { ...item, index } : state.primary,
```

Comment:
We can now say isRadio=false for the split button and it will work. Yay! I guess this was another fix that I forgot to put in my git commit message.

Reply:
Thats useful to have.

Reply:
Thanks.